README.md

```diff
@@ -130,6 +130,7 @@ Developer experience first, extremely flexible code structure and only keep what
 - ☂️ Code coverage with [Codecov](https://about.codecov.io/codecov-free-trial/?utm_source=github&utm_medium=paid-community&utm_campaign=general-fy25q1-nextjs&utm_content=github-banner-nextjsboilerplate-logo)
 - 📝 Logging with Pino.js and Log Management with [Better Stack](https://betterstack.com/?utm_source=github&utm_medium=sponsorship&utm_campaign=next-js-boilerplate)
 - 🖥️ Monitoring as Code with [Checkly](https://www.checklyhq.com/?utm_source=github&utm_medium=sponsorship&utm_campaign=next-js-boilerplate)
+- 🔐 Security and bot protection ([Arcjet](https://launch.arcjet.com/Q6eLbREo))
```

Comment:
Found a small error in the URL, at the end of the link there is a trailing o.

I'll take care of it.

Reply:
Thanks. It should be https://launch.arcjet.com/Q6eLbRE